runit/run-service-dir.patch

```diff
@@ -0,0 +1,11 @@
+--- ./sv.c.orig
++++ ./sv.c
+@@ -32,7 +32,7 @@
+ char *progname;
+ char *action;
+ char *acts;
+-char *varservice ="/service/";
++char *varservice ="/etc/service/";
```

Comment:
why?  it's usually `/service` or `/sv` when using djb-style supervision trees.

Reply:
good catch, I blindly took this from alpine, thinking `calico` also hardcoded it, but it looks like calico is setting it using `SVDIR`.

I removed this patch